services/likes_service.py

```
@@ -3,14 +3,16 @@
 from models.like import LikeCreate, LikeResponse
 from repositories.likes_repo import get_likes, save_likes
 
-def toggle_like_service(user_id: str, like_in: LikeCreate):
+def toggle_like_service(user_id: str , like_in: LikeCreate):
     likes = get_likes()
+
     existing_like = next((l for l in likes if l["user_id"] == user_id
```

Ruff (0.14.13)

Ambiguous variable name: `l`

(E741)